lib/platform.h

```diff
@@ -100,6 +100,9 @@ namespace cppcheck {
 
         char defaultSign;  // unsigned:'u', signed:'s', unknown:'\0'
 
+        bool windows{false}; // indicates if the platform is Windows
+        bool win_ansi{false}; // indicates if Windows should use the "<func>A()" ANSI variants
```

Comment:
I was thinking about making this an `enum` but I am not sure if there is anything else beside the `*W` and `*A` we would tackle here.